tests/core/test_accounts.py

```diff
@@ -54,18 +54,25 @@
 ]
 
 
-PRIVATE_KEY_AS_BYTES = b'unicorns' * 4
-PRIVATE_KEY_AS_BYTES_ALT = b'rainbows' * 4
```

Comment:
Oh no, why did you kill the unicorns and rainbows? :cry:

I may revive them later... For now, I'll just merge as is.

Reply:
My main issue was generating a `hexstr` from those :/

Reply:
I think we can generate a `PrivateKey` from those and get the `hexstr` from that 🤔

Reply:
The hexstr for them was in my comment asking about testing int and hexstrs.

Reply:
So the block could be:

```py
PRIVATE_KEY_AS_BYTES = b'unicorns' * 4
PRIVATE_KEY_AS_HEXSTR = '0x756e69636f726e73756e69636f726e73756e69636f726e73756e69636f726e73'
PRIVATE_KEY_AS_INT = 0x756e69636f726e73756e69636f726e73756e69636f726e73756e69636f726e73
PRIVATE_KEY_AS_OBJ = keys.PrivateKey(PRIVATE_KEY_AS_BYTES)
ACCT_ADDRESS = '0xa79F6f349C853F9Ea0B29636779ae3Cb4E3BA729'
```

Reply:
Yeah, I meant the account address, not the `hexstr`. Generated a `PrivateKey` from the bytes and got address from the account. 👍 Added them back!

Reply:
I'll remove the casting and represent the int as the primitive.